Replication Server 6.2.19 release (#3423)
EnterpriseDB/docs: product_docs/docs/eprs/6.2/03_installation/installing_jdbc_driver.mdx

- **PostgreSQL**, use the PostgreSQL JDBC driver. To download the latest supported driver (Java 8), see the [JDBC drivers section](https://jdbc.postgresql.org/download/) on the PostgreSQL Downloads page. 

- **Oracle**, use the freely available [Oracle JDBC](https://www.oracle.com/database/technologies/appdev/jdbc-downloads.html) driver.

- **Microsoft SQL Server**, use the freely available jTDS driver.

## Configuring the driver

After downloading the driver, create a symlink in the `XDB-install-folder/lib/jdbc` directory that points to the location where you installed the driver. 

For Linux, create a symlink for the driver using these naming conventions:

   |        Driver        |  symlink name  |
   | -------------------- | -------------- |
   | EDB                  | edb-jdbc18.jar |
   | PostgreSQL           | postgresql.jar |
   | Oracle               | ojdbc.jar      |
   | jTDS                 | jtds.jar       |

 For interactive installers, the drivers need to be copied to the `XDB-install-folder/lib/jdbc` directory and renamed using the same naming conventions.

## Retaining the driver in an upgrade 

> When upgrading from a previous version of Replication Server, the already shipped or placed drivers in `XDB-install-folder/lib/jdbc` are retained. Check that the retained drivers are supported below:

Reviewer: ```suggestion
When upgrading from a previous version of Replication Server, any drivers in `XDB-install-folder/lib/jdbc` are retained. Check that the retained drivers are at the minimum supported or later version:
```
